core-bundle/src/Resources/contao/classes/Frontend.php

```diff
@@ -323,7 +323,7 @@ public static function getRootPageFromUrl()
 		$accept_language = Environment::get('httpAcceptLanguage');
 
 		// Get the language from the URL if it is not set (see #456)
-		if (!isset($_GET['language']) && Config::get('addLanguageToUrl'))
+		if (!isset($_GET['language']) && $GLOBALS['TL_CONFIG']['addLanguageToUrl'])
```

Comment:
This seems wrong to me. Why are you not using `contao.prepend_locale` here? If we still need the global variable, we should un-deprecate it and continue to use `Config::set()` IMHO.

Reply:
You are right, I think I mixed up `addLanguageToUrl` with legacy routing. Will fix.

Reply:
Fixed in 632af669fd0850f6547e6012d81cdbf5c179780f